towerlib/entities/project.py

```diff
@@ -530,3 +547,23 @@ def custom_virtualenv(self, value):
         else:
             raise InvalidValue(f'{value} is invalid. Condition max_characters must be less than or equal to '
                                f'{max_characters}')
+
+    @property
+    def update(self):
+        """Update the ansible tower project with given project id.
+
+        Returns:
+            list: List of response of api request as json on success, False otherwise.
+
+        """
+        update_url = '{api}/projects/{id}/update/'.format(api=self._tower.api, id=self.id)
+        response = self.session.post(update_url)
+        if not response.ok:
+            self._logger.error(
+                "Error updating the project '{}'. response was: {})".format(self.name, response.text))
+            return None
+        else:
+            self._logger.info("The project '{}' was successfully updated with the latest scm version."
+                              .format(self.name))
+            return response.json()
```

Comment:
Could you please explain in which file I should place this class and maybe give an example how I could use them?

Reply:
This was a suggestion based on our discussions. I am not sure if it is still valid because you removed all kinds of things that was client business logic. Should we maybe have a chat about the usage case again and whether it still makes sense? I think a syncing would be good around now :)